### PR TITLE
NO-TICKET - Minor improvements for token services

### PR DESCRIPTION
**Pull request by 1sakov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/

## Description of Work

[ what work was done in this ticket ]

## Screenshots (Optional)

| Screenshot #1 | Screenshot #2 | Screenshot #3 |
|:-------------:|:-------------:|:-------------:|
| [Paste Pic 1] | [Paste Pic 2] | [Paste Pic 3] |

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletPresenter.kt [hunk 1]
```diff
@@ -24,58 +27,69 @@ class WalletPresenter(
...
+        view?.showRefreshing(isRefreshing = newState.isLoading())
+
+        when (newState) {
+            is UserTokensState.Idle -> Unit
+            is UserTokensState.Loading -> Unit
+            is UserTokensState.Refreshing -> Unit
+            is UserTokensState.Error -> view?.showErrorMessage(newState.cause)
+            is UserTokensState.Empty -> view?.showBalance(null)
+            is UserTokensState.Loaded -> {
+                val usdc = newState.solTokens.find { it.isUSDC }
```
Comment (kabramovp2p) [inline app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletPresenter.kt:65]: but we should get all tokens with needed extras not only USDC :(

Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletPresenter.kt:65]: isUsdc

**Review by kabramovp2p — APPROVED** [review]

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletPresenter.kt (hunk 1 above)
Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletPresenter.kt:65]: For crypto, right?
For Wallet you need only usdc

Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletPresenter.kt:65]: why? 
The result of `find` is token, not boolean

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletPresenter.kt:65]: { it.isUSDC } -> { it.isUsdc } :)

**Pull request merged**